packages/cache-core/src/cacheHandler.ts

```diff
@@ -97,15 +104,15 @@ export class Cache implements CacheHandler {
     return false
   }
 
-  async get(cacheKey: string): Promise<CacheEntry | null> {
+  async get(pageKey: string): Promise<CacheEntry | null> {
     try {
-      if (Cache.noCacheRoutes.includes(cacheKey)) {
+      if (Cache.noCacheRoutes.includes(pageKey)) {
         return null
       }
 
-      Cache.logger.info(`Reading cache data for ${cacheKey}`)
+      Cache.logger.info(`Reading cache data for ${pageKey}`)
 
-      const data = await Cache.cache.get(this.getPageCacheKey(cacheKey), {
+      const data = await Cache.cache.get(pageKey, this.getPageCacheKey(pageKey), {
```

Comment:
Or maybe here it makes sense to add cache

Reply:
this is a core functionality which should not be altered, you can define just what cache strategy you want to use (like, memory, redis, s3, .etc)